packages/inventory-compliance/src/SystemRulesTable.js

```diff
@@ -239,8 +239,8 @@ class SystemRulesTable extends React.Component {
     )
 
     referencesList = (references) => references.reduce((acc, reference, i) => ([
-        acc, ', ', this.conditionalLink(reference.label, reference.href, { target: '_blank', key: i + 1 })
-    ]), this.conditionalLink(references[0].label, references[0].href, { target: '_blank', key: 0 }));
+        acc, i !== 0 ? ', ' : '', this.conditionalLink(reference.label, reference.href, { target: '_blank', key: i + 1 })
```

Comment:
Is this correct? Beacause if I try it on sample array it looks kinda strange
```JS
['first', 'second', 'third'].reduce((acc, ref, i) => ([acc, i !== 0 ? ', ' : '', ref]), [])
// output is
[
 [ [ [], '', 'first' ], ', ', 'second' ],
 ', ',
 'third'
]
```
I guess you want to have flat array
```JS
referencesList = (references) => references.reduce((acc, reference, i) => ([
  ...acc,
  this.conditionalLink(reference.label, reference.href, { target: '_blank', key: i + 1 })
])
```

Reply:
@dLobatog ^

Reply:
Fixed, thanks! I guess this wasn't noticed because on the UI once it's cast to a string you don't see it